docs/the-branching-workflow.md

```diff
@@ -0,0 +1,33 @@
+# The Branching Workflow
+
+![The Branching Workflow](images/the-branching-workflow.png)
```

Comment:
Now you are linking to a file that doesn't exist?

Reply:
Have another look.